cup.css

```
@@ -0,0 +1,56 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    background: #262626;
+}
+body{
+    height: 100vh;
+    width: 100%;
+    display: grid;
+    place-items: center;
+}
+
+.cup{
+    width: 200px;
+    height:220px;
+    
+    border: 6px solid #fff;
+    border-top: 2px solid transparent;
+    border-radius: 15px;
+    border-top-left-radius: 5px;
+    border-top-right-radius: 5px;
+    
+    box-shadow: 0 0 0 6px #fff , 0 20px 35px rgba(0,0,0,1);
+
+    background: url(https://image.ibb.co/fmHm66/wave.png);
+    background-repeat: repeat-x;
+    
```

```suggestion
    background-size:  cover;
```
suggestion number (1 of 2) to create smooth animation